matterpy/contrib/rss.py

```diff
@@ -15,39 +16,50 @@ def init(manager, conf):
     _mgr = manager
 
     loop = asyncio.get_event_loop()
-    rss_feeds = [rss_url for rss_url in _conf['rss_feed'].split(',')]
-    for rss_url in rss_feeds:
-        loop.create_task(post_rss(rss_url))
+    feeddict = defaultdict(dict)
 
+    for i in _conf:
+        if '.' in i:
+            keys = i.split('.')
+            feeddict[keys[1]][keys[2]] = _conf.get(i)
+    for i in feeddict:
+        loop.create_task(post_rss(feeddict.get(i)))
 
-async def post_rss(rss_url):
+
+async def post_rss(feedinfo):
     n = 0
     last_message = ''
-    d = feedparser.parse(rss_url)
+    fi = feedinfo
+    d = feedparser.parse(fi.get('url'))
     while True:
         try:
             for i in d.entries:
```

Comment:
Did you check what's in `i`? This could simplify your code below. No need to count up using the `n` variable etc